19.03.5.sh

```diff
@@ -284,6 +284,9 @@ do_install() {
 	if [ -z "$lsb_dist" ] && [ -r /etc/redhat-release ]; then
 		lsb_dist='redhat'
 	fi
+	if [ -z "$lsb_dist" ] && [ -r /etc/SuSE-release ]; then
```

Comment:
We can probably drop this, as this is deprecated and might be removed (even in Service Pack):

```
# cat /etc/SuSE-release
SUSE Linux Enterprise Server 12 (x86_64)
VERSION = 12
PATCHLEVEL = 5
# This file is deprecated and will be removed in a future service pack or release.
# Please check /etc/os-release for details about this release.
```

The fallback to `os-release` should work.

Reply:
Updated in latest commit!